package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@linzjs/argo-tasks",
   "private": true,
-  "version": "4.12.0",
```

Comment:
Why we downgrade to `4.11,0`?

Reply:
werid, looks to have been a bad merge somewhere.

Reply:
Have fixed this now.